src/inc/DefaultSettings.h

```diff
@@ -26,6 +26,9 @@ constexpr COLORREF DEFAULT_BACKGROUND_WITH_ALPHA = OPACITY_OPAQUE | DEFAULT_BACK
 constexpr COLORREF POWERSHELL_BLUE = RGB(1, 36, 86);
 
 constexpr short DEFAULT_HISTORY_SIZE = 9001;
+
+#pragma warning(push)
+#pragma warning(disable : 26426)
 const std::wstring DEFAULT_FONT_FACE{ L"Consolas" };

```

Comment:
@miniksa Can you make the strings constexpr instead of disabling the warning? 
constexpr wchar_t DEFAULT_FONT_FACE[] = L"Consolas";  
or 
constexpr auto DEFAULT_FONT_FACE[] = L"Consolas";  

Not sure how these are used but I think this would work.

Reply:
I was intending to fix this by making them std::wstring_view, but that cascaded into other things. So I then had intended to file #2674 and fix this up in another step. They're disabled for now, but I don't intend them to be permanently.